src/modules/coin/schemas.rs

```diff
@@ -90,14 +90,22 @@ pub struct FtContractMetadataResponse {
 pub struct Coin {
     /// "nearprotocol" for NEAR, "nep141" for FT
     pub standard: String,
-    pub balance: types::U128,
+    /// For NEAR/FT, there is always 1 item in the list. For MT, there could be more
+    pub balances: Vec<BalanceItem>,
```

Comment:
It could be `Map<String, u128>`, but I decided to be more explicit. In the current solution, the user knows the field names